cmd/restic/cmd_generate.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var cmdGenerate = &cobra.Command{
-	Use:   "generate [command]",
+	Use:   "generate [flags] [command]",
```

Comment:
I don't see any commands which are accepted by `generate`. Please remove the `[command]` part.

Reply:
Done!